MinecraftClient/McClient.cs

```diff
@@ -3154,6 +3154,22 @@ public void OnDestroyEntities(int[] Entities)
         /// <param name="Dy"></param>
         /// <param name="Dz"></param>
         /// <param name="onGround"></param>
+        public void OnEntityPosition(int EntityID, Double Dx, Double Dy, Double Dz, byte _yaw, byte _pitch, bool onGround)
+        {
+            if (entities.ContainsKey(EntityID))
+            {
+                Location L = entities[EntityID].Location;
+                L.X += Dx;
+                L.Y += Dy;
+                L.Z += Dz;
+                entities[EntityID].Location = L;
+                entities[EntityID].Yaw = _yaw * (1F / 256) * 360;
+                entities[EntityID].Pitch = _pitch * (1F / 256) * 360;
```

Comment:
The yaw and pitch calculation should be done in `Protocol18` before passing to handler so that:
- Client handler don't need to care about protocol (e.g. how to convert the `byte` back to yaw and pitch)
- If there's some update to the protocol (data type or calculation change), client handler can remain untouched